changelogs/fragments/448-acme-request-timeouts.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - acme_* modules - Add parameter ``request_timeout`` to manage HTTP(S) request timeout (https://github.com/ansible-collections/community.crypto/issues/447, https://github.com/ansible-collections/community.crypto/pull/448).
```

Just a tiny thing that got missing in the last change :)
```suggestion
  - acme_* modules - add parameter ``request_timeout`` to manage HTTP(S) request timeout (https://github.com/ansible-collections/community.crypto/issues/447, https://github.com/ansible-collections/community.crypto/pull/448).
```

---

Ah, I thought that was a typo :sweat_smile: 
Sentences start with a Capital and end with a punctuation mark =P

Anyhow, lowercased it :)

---

Sentences do, but these are not proper sentences :) I don't know who came up with this, but since it was the convention in ansible/ansible for quite some time I'm trying to make sure that all fragments look the same way. Makes the changelog easier to read if such things don't change from entry to entry ;-) (https://docs.ansible.com/ansible/devel/community/collection_development_process.html#changelog-fragment-entry-format)